### DISCUSSION TITLE
Media player icons in groups sections

### DISCUSSION THREAD

**Discussion by anes300** [punxaphil/custom-sonos-card#137] [Q&A] [answered]
Hi @johanfrick,

I have a question in regards to the icons next to the media players that are shown in your examples within the README. How do you get these into the card? I had a look into the code, but TS is not my strong side, so I can't seem to figure out how the icon element is set properly. Can you help me out? 😊

<img width="210" alt="image" src="https://github.com/johanfrick/custom-sonos-card/assets/54175142/1757958f-00c7-483f-b579-2ae388efbe9d">

Once again, thank you for your great work with the V6 card!

Thank you in advance,
Anes Skrijelj

**Comment by punxaphil** [comment, marked as answer]
1. Install this one in HACS [GitHub - elax46/custom-brand-icons: Custom brand icons for Home Assistant 10](https://github.com/elax46/custom-brand-icons)
2. In your Home Assistant go to Settings
3. entities
4. select your media player
5. Click the cogwheel
6. Update icon field

---

[![BMC|170x37, 75%](https://community-assets.home-assistant.io/original/4X/c/b/3/cb32273d8710d5efb2561ef840e68c2fe1b61d52.png) ](https://www.buymeacoffee.com/punxaphil)

**Reply by anes300** [reply]
Brilliant, thank you!
